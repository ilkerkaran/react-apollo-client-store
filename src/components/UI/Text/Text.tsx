import React from 'react';

const Text = ({ text = undefined }:{text:string | undefined }) => (

  <p>{text}</p>

);

export default Text;
