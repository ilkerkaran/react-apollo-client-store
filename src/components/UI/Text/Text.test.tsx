import React from 'react';
import renderer from 'react-test-renderer';

import Text from './Text';

describe('<Text /> snapshot ', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Text text="some text" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
