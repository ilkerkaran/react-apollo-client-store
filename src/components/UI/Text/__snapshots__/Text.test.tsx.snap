// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> snapshot  renders correctly 1`] = `
<p>
  some text
</p>
`;
